runtime/src/bank.rs

```diff
@@ -4048,11 +4048,29 @@ impl Bank {
         {
             let block_cost_limit = simd_0286_block_limits();
             let mut cost_tracker = self.write_cost_tracker().unwrap();
-            let account_cost_limit = cost_tracker.get_account_limit();
+            let account_cost_limit = if self
+                .feature_set
+                .is_active(&feature_set::raise_account_cu_limit::id())
+            {
+                (block_cost_limit as f64 * 0.4) as u64
+            } else {
+                cost_tracker.get_account_limit()
+            };
             let vote_cost_limit = cost_tracker.get_vote_limit();
             cost_tracker.set_limits(account_cost_limit, block_cost_limit, vote_cost_limit);
         }
 
+        if self
+            .feature_set
+            .is_active(&feature_set::raise_account_cu_limit::id())
+        {
+            let mut cost_tracker = self.write_cost_tracker().unwrap();
+            let block_cost_limit = cost_tracker.get_block_limit();
+            let vote_cost_limit = cost_tracker.get_vote_limit();
+            let account_cost_limit = (block_cost_limit as f64 * 0.4) as u64;
```

Comment:
same code as repeated above - we don't need both.

ONLY this one should exist, right?

Reply:
The problem happens if we activate 306 before 286:

1. activate 306
2. account limit goes 12M --> 24M
3. 286 activated
4. block limit goes 60M --> 100M
5. (account limit stays at 24M)

Need a kick after step 4 to say "306 is activated and you changed the value from which we derive the account limit so recompute it"